SWLOR.Game.Server/Feature/ShipModuleDefinition/CapitalWeaponsModuleDefinition.cs

```diff
@@ -0,0 +1,270 @@
+            CapitalWeapons("cap_weps2", "Tier 2 Capship Weapons", "T2CapWeapons", "Weapons assigned to Tier 2 Capital Ships.", 26);
+            CapitalWeapons("cap_weps3", "Tier 3 Capship Weapons", "T3CapWeapons", "Weapons assigned to Tier 3 Capital Ships.", 32);
+            CapitalWeapons("cap_weps4", "Tier 4 Capship Weapons", "T4CapWeapons", "Weapons assigned to Tier 4 Capital Ships.", 38);
+            CapitalWeapons("cap_weps5", "Tier 5 Capship Weapons", "T5CapWeapons", "Weapons assigned to Tier 5 Capital Ships.", 44);
+            CapitalWeapons("cap_weps6", "Tier 6 Capship Weapons", "T6CapWeapons", "Weapons assigned to Corvettes.", 50);
+            CapitalWeapons("cap_weps7", "Tier 7 Capship Weapons", "T7CapWeapons", "Weapons assigned to Frigates.", 56);
+            CapitalWeapons("cap_weps8", "Tier 8 Capship Weapons", "T8CapWeapons", "Weapons assigned to Cruisers.", 60);
+            CapitalWeapons("cap_weps9", "Tier 9 Capship Weapons", "T9CapWeapons", "Weapons assigned to Heavy Cruisers.", 64);
+            CapitalWeapons("cap_weps10", "Tier 10 Capship Weapons", "T10CapWeapons", "Weapons assigned to Battlecruisers.", 67);
+            CapitalWeapons("cap_weps11", "Tier 11 Capship Weapons", "T11CapWeapons", "Weapons assigned to Battleships.", 70);
+            CapitalWeapons("cap_weps12", "Tier 12 Capship Weapons", "T12CapWeapons", "Weapons assigned to Dreadnoughts.", 75);
+
+            return _builder.Build();
+        }
+
+        private void CapitalWeapons(
+            string itemTag,
+            string name,
+            string shortName,
+            string description,
+            int dmg)
+        {
+            _builder.Create(itemTag)
+                .Name(name)
+                .ShortName(shortName)
+                .Description(description)
+                .Type(ShipModuleType.CapitalWeapons)
+                .Texture("iit_ess_079")
+                .MaxDistance(50f)
+                .ValidTargetType(ObjectType.Creature)
+                .PowerType(ShipModulePowerType.High)
+                .RequirePerk(PerkType.OffensiveModules, 1)
+                .Recast(2f)
+                .Capacitor(5)
+                .ActivatedAction((activator, activatorShipStatus, target, targetShipStatus, moduleBonus) =>
+                {
+                    var attackRoll = Random.D10(1);
+                    var attackType = "";
+                    var attackBonus = activatorShipStatus.ThermalDamage;
+                    var attackerStat = GetAbilityScore(activator, AbilityType.Perception);
+                    var attack = Stat.GetAttack(activator, AbilityType.Perception, SkillType.Piloting, attackBonus);
+                    var defenseBonus = targetShipStatus.ThermalDefense * 2;
+                    var defense = Stat.GetDefense(target, CombatDamageType.Thermal, AbilityType.Vitality, defenseBonus);
+                    if (attackRoll < 7)
+                    {
+                        attackType = "laser";
```

Comment:
Strings are very error prone and difficult to maintain. Please set up an enumeration - you can make it private within this class. I.E:

private enum AttackType
{
    Invalid = 0,
    Laser = 1,
...etc
}